content/shared/influxdb3-cli/test/schedule_plugin.md

```diff
@@ -0,0 +1,60 @@
+
+The `influxdb3 test schedule_plugin` command tests a schedule plugin. Use this command to verify plugin behavior without creating a trigger.
+
+## Usage
+
+<!--pytest.mark.skip-->
+
+```bash
+influxdb3 test schedule_plugin [OPTIONS] --database <DATABASE_NAME> <FILENAME>
+```
+## Arguments
+
+- **PLUGIN_NAME**: Path to the plugin file on the server. Use the full relative path, such as `<plugin-dir>/<plugin-file-name>.py`.
+
+## Options
+
+| Option | Flag               | Description                                                                              |
+| :----- | :----------------- | :--------------------------------------------------------------------------------------- |
+| `-H`   | `--host`           | Host URL of the running {{< product-name >}} server (default: `http://127.0.0.1:8181`)   |
+| `-d`   | `--database`       | _({{< req >}})_ Name of the database to operate on                                       |
+|        | `--token`          | _({{< req >}})_ Authentication token                                                     |
+|        | `--tls-ca`         | Path to a custom TLS certificate authority (for testing or self-signed certificates)     |
+| `-h`   | `--help`           | Print help information                                                                   |
+|        | `--help-all`       | Print detailed help information                                                          |
+
```

Comment:
Missing options. Make sure you have the latest version and then run `> influxdb3 test schedule_plugin --help`